packages/smithy-http/src/smithy_http/aio/aiohttp.py

```diff
@@ -80,7 +80,7 @@ async def send(
         headers_list = list(
             chain.from_iterable(
                 fld.as_tuples()
-                for fld in request.fields.get_by_type(FieldPosition.HEADER)
+                for fld in request.fields
```

Comment:
Why was this change made?

Reply:
We're not actually adding the FieldPosition correctly for headers so we end up with an empty list here. I have a task to go back and work out what's failing. This was a carry over from the original client POC that I still need to sort out.